### DISCUSSION TITLE
Expander Header Pointerover issue

### DISCUSSION THREAD

**Discussion by iMrShadow** [AvaloniaUI/Avalonia#16408] [Q&A] [answered]
I am trying to change the Expander's header when I hover it over with my mouse. However, I have issues finding the [right template style selector](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/Expander.xaml). 

The closest thing I got to changing the background colour is with

```xaml
<Style Selector="Expander/template/ToggleButton:pointerover/template/ContentPresenter">
	<Setter Property="Background"
            Value="#ff0000" />
</Style>
```
![image](https://github.com/user-attachments/assets/83dbcfb5-0fb4-4100-8d28-4063ac05244e)

Thanks

**Comment by maxkatz6** [comment, marked as answer]
This should be the right one:
`Selector="Expander /template/ ToggleButton:pointerover /template/ Border#ToggleButtonBackground"`

**Reply by AndrewSSB** [reply]
But how do you change the "normal" background color of the header ?

<img width="236" height="67" alt="image" src="https://github.com/user-attachments/assets/c55c0858-0773-41e4-8c33-2d0d8774365d" />
